README.md

```diff
@@ -1,6 +1,38 @@
 # 🌟 Azure MCP Server
 
-The Azure MCP Server implements the [MCP specification](https://modelcontextprotocol.io) to create a seamless connection between AI agents and Azure services.  Azure MCP Server can be used alone or with the [GitHub Copilot for Azure extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azure-github-copilot) in VS Code.  This project is in Public Preview and implementation may significantly change prior to our General Availability.
+[![npm version](https://img.shields.io/npm/v/@azure/mcp.svg)](https://www.npmjs.com/package/@azure/mcp)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![GitHub issues](https://img.shields.io/github/issues/Azure/azure-mcp.svg)](https://github.com/Azure/azure-mcp/issues)
+
+The Azure MCP Server implements the [MCP specification](https://modelcontextprotocol.io) to create a seamless connection between AI agents and Azure services. Azure MCP Server can be used alone or with the [GitHub Copilot for Azure extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azure-github-copilot) in VS Code.
+
+> **🚀 New in v0.5.0:** Namespace mode (default), improved tool organization, and Azure Workbooks support
+> **📢 Public Preview:** This project is in Public Preview - implementation may significantly change prior to General Availability.
+
+## 📋 Table of Contents
+
+- [🚀 Quick Start](#️-quick-start)
+- [⚙️ VS Code Install Steps](#️-vs-code-install-steps-recommended)
+- [🤖 Available Azure MCP Servers](#-available-azure-mcp-servers)
+- [▶️ Getting Started](#️-getting-started)
+- [✨ What can you do with the Azure MCP Server?](#-what-can-you-do-with-the-azure-mcp-server)
+- [🛠️ Currently Supported Tools](#️-currently-supported-tools)
+- [📖 Complete Tools Documentation](#-complete-tools-documentation)
+- [🔄️ Upgrading Existing Installs](#️-upgrading-existing-installs-to-the-latest-version)
+- [⚙️ Advanced Install Scenarios](#️-advanced-install-scenarios-optional)
+- [📝 Troubleshooting](#-troubleshooting)
+- [❓ Frequently Asked Questions](#-frequently-asked-questions)
+- [👥 Contributing](#-contributing)
+
+## 🚀 Quick Start
+
+**Get started in under 60 seconds:**
+
+1. **Install VS Code** and **GitHub Copilot** extensions ([VS Code](https://code.visualstudio.com/download) + [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) + [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat))
+2. **Install Node.js 20+** with `npx` in your PATH ([Download Node.js](https://nodejs.org/en/download))
```

Comment:
could you rebase this PR on the latest `main` branch?  We deleted the bideo below and changed the quick start to point people to the VS Code Marketplace VSIX instead of the NPX package